addons/l10n_sa_edi/models/account_journal.py

```diff
@@ -85,6 +85,17 @@ class AccountJournal(models.Model):
     l10n_sa_latest_submission_hash = fields.Char("Latest Submission Hash", copy=False,
                                                  help="Hash of the latest submitted invoice to be used as the Previous Invoice Hash (KSA-13)")
 
+    def _l10n_sa_reset_chain_head_error(self, trigger_webservice=False):
```

Comment:
```suggestion
    def _l10n_sa_reset_chain_head_error(self):
```
And later just call `action_retry_edi_documents_error` with `trigger_webservice=False`. This will remove the some redundancy.